src/server_resolver.rs

```diff
@@ -314,8 +312,8 @@ impl MatrixConnector {
     /// Create new [`MatrixConnector`] with the given [`MatrixResolver`].
     pub fn with_resolver(resolver: MatrixResolver) -> MatrixConnector {
         let client_config = ClientConfig::builder()
-            .with_safe_defaults()
```

Comment:
> `ConfigBuilder::with_safe_defaults` - calls to this can simply be deleted since safe defaults are now implicit.
>
> https://github.com/rustls/rustls/releases/tag/v%2F0.22.0